packages/fields/src/fields/template-title/index.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import type { Field } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import type { Template } from '../../types';
+import { getItemTitle } from '../../actions/utils';
+import TitleView from '../title/view';
+
+const templateTitleField: Field< Template > = {
+	label: __( 'Template' ),
+	id: 'title',
+	getValue: ( { item } ) => getItemTitle( item ),
+	render: TitleView,
+	enableHiding: false,
+	enableGlobalSearch: true,
```

Comment:
I've noticed `enableGlobalSearch` is not enabled for the `titleField` 🤔 

Also, don't forget adding the `type: text`. I'm already regretting adding the fallback for discovering how to sort fields that don't have the type: it makes the type redundant :)

Reply:
> I've noticed `enableGlobalSearch` is not enabled for the `titleField` 🤔 

This is because we don't search the pages client-site while we do it for templates.

Reply:
Exactly. I think it doesn't hurt to have it there though because consumers could do load posts and do the filtering client side.

Regarding the `type` isn't it also used in DataForms for providing editing UI? If yes, the template title shouldn't be editable by default, as you can't rename all templates.